mailpoet/lib/Newsletter/ViewInBrowser/ViewInBrowserRenderer.php

```diff
@@ -93,19 +93,15 @@ public function render(
     return $renderedNewsletter;
   }
 
-  private function prepareShortcodes($newsletter, $subscriber, $queue, $wpUserPreview) {
-    if ($queue instanceof SendingQueueEntity) {
-      $this->shortcodes->setQueue($queue);
-    }
-
-    if ($newsletter instanceof NewsletterEntity) {
-      $this->shortcodes->setNewsletter($newsletter);
-    }
-
+  private function prepareShortcodes(
```

Comment:
I'm also really convinced defining this method makes much sense since it's not really adding much, keeping it though, so that when we refactor how data is passed to the shortcodes ([MAILPOET-4087]) service we can change this.